pkg/limayaml/validate_test.go

```diff
@@ -48,6 +48,31 @@ func TestValidateProbes(t *testing.T) {
 	assert.Error(t, err, "field `probe[0].script` must start with a '#!' line")
 }
 
+func TestValidateAdditionalDisks(t *testing.T) {
+	images := `images: [{"location": "/"}]`
+
+	validDisks := `
+additionalDisks:
+  - name: "disk1"
+  - name: "disk2"
```

Comment:
Does not really matter, but our `.yamllint` style does not indent sequences. 

```suggestion
additionalDisks:
- name: disk1
- name: disk2
```